src/edit/InstanceForm.js

```diff
@@ -222,6 +224,15 @@ class InstanceForm extends React.Component {
 
     this.cViewMetaData = this.props.stripes.connect(ViewMetaData);
     this.accordionStatusRef = createRef();
+    this.state = { isInstanceSetForDeletion: false };
+  }
+
+  componentDidMount() {
+    const { initialValues } = this.props;
+
+    if (initialValues?.deleted) {
+      this.setState({ isInstanceSetForDeletion: true });
+    }
   }
```

Comment:
It can be done in initial state

```
this.state = {
 isInstanceSetForDeletion: false,
 isInstanceSetForDeletion: Boolean(props.initialValues?.deleted),
  };
```